src/lib/gundb/UserStorageClass.js

```diff
@@ -1896,12 +1884,13 @@ export class UserStorage {
       const groupValue = await gunGroupIndexValue.then()
       logger.debug('extractProfiler result :', { group, value, groupValue })
 
-      if (!isNull(groupValue) && Gun.node.is(groupValue)) {
+      if (groupValue != null && groupValue.profile) {
         return {
           gunProfile: gunGroupIndexValue.get('profile'),
         }
       }
       logger.warn('_extractProfileToShow invalid profile', { group, value, groupValue })
+      return undefined
```

Comment:
return without argument returns undefined
if it's last statement in the function body - you could omit it at all

```suggestion
```